library/public/image.h

```diff
@@ -18,6 +18,22 @@ namespace f3d
 class F3D_EXPORT image
 {
 public:
+  /**
+   * Enumeration of supported export formats
+   * =======================================
+   * PNG: Supports channel size up to 2 bytes
+   * JPG: Supports channel size of 1 byte
+   * TIF: Supports channel size up to 4 bytes
+   * BMP: Supports channel size of 1 byte
+   */
+  enum class Format : unsigned char
+  {
+    PNG,
+    JPG,
+    TIF,
+    BMP
+  };
+
   /**
    * Create an image from file, the following formats are supported:
    * PNG, PNM, TIFF, BMP, HDR, JPEG, GESigna, MetaImage, TGA.
```

Comment:
I dont this that list is up to date ?

Reply:
plait-il?

Reply:
I dont **think** that list is up to date ?

Reply:
what about EXR ?